rsconnect/api.py

```diff
@@ -354,8 +354,8 @@ def app_access(self, app_guid: str) -> None:
         if self.is_app_failed_response(response):
             raise RSConnectException(
                 "Could not access the deployed content. "
-                + "The app might not have started successfully. "
-                + "Visit it in Connect to view the logs."
+                + "The app might not have started successfully."
+                + f"\n\tFor more information: {self.app_config(app_guid)['logs_url']}"
```

Comment:
New line + tab to match the other URLs that get put in the output, but that's not strictly necessary.